api/proto/common/v2/common_v2.proto

```diff
@@ -3,21 +3,39 @@ package common.v2;
 import "common/common.proto";
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common/v2";
 
-
+// BlobHeader contains the information needed to disperse a blob to the EigenDA network.
+// It can be thought of as an "eigenDA tx", in that it plays a purpose similar to an eth_tx to disperse a 4844 blob.
```

Comment:
"It" means BlobHeader? I think Blob (which is different than BlobHeader) is analogous to a txn

Reply:
No the blob is the payload. the blobheader is the tx, specifying price, commitment, etc.

Reply:
Now I don't know what is blob/blobheader/payload/txn... https://en.wikipedia.org/wiki/Semantic_satiation

Reply:
I switched the wording as follows. Note that this paragraph moved into `DisperseBlobRequest` (as requested by a prior comment).

```
  // This header can be thought of as an "eigenDA tx", in that it plays a purpose similar to an eth_tx to disperse a
  // 4844 blob. Note that a call to DisperseBlob requires the blob and the blobHeader, which is similar to how
  // dispersing a blob to ethereum requires sending a tx whose data contains the hash of the kzg commit of the blob,
  // which is dispersed separately.
```